Client/ViewController.swift

```diff
@@ -28,20 +29,30 @@ class ViewController: UIViewController {
         let userService = Karhoo.getUserService()
         userService.logout().execute(callback: { _ in})
         
-        userService.login(userLogin: UserLogin(username: Keys.userServiceEmailKarhoo,
-                                               password: Keys.userServicePasswordKarhoo)).execute(callback: { result in
+        userService.login(userLogin: UserLogin(username: Keys.userServiceEmailAdyen,
```

Comment:
Perhaps we can hide the configuration from this class. I mean if we have a variable in Keys which we configure it based on our preference, then you don't need to change this class in order to switch between `userServiceEmailAdyen` or `userServiceEmailKarhoo` 

in Keys class:
`var userServiceEmail = userServiceEmailAdyen`

then in this class you always use `Keys.userServiceEmail`